test/libsolidity/SemanticTest.h

```diff
@@ -80,10 +80,11 @@ class SemanticTest: public SolidityExecutionFramework, public EVMVersionRestrict
 private:
 	TestResult runTest(std::ostream& _stream, std::string const& _linePrefix, bool _formatted, bool _isYulRun, bool _isEwasmRun);
 	bool checkGasCostExpectation(TestFunctionCall& io_test, bool _compileViaYul) const;
-	void initializeBuiltins();
+	std::map<std::string, Builtin> makeBuiltins();
```

Comment:
```suggestion
	std::map<std::string, Builtin> makeBuiltins() const;
```

Reply:
Some builtins are currently calling non-const functions, e.g. `balance` -> `ExecutionFramework::balanceAt(address)` and `storageEmpty` -> `ExecutionFramework::storageEmpty(m_contractAddress)`. But it looks like that they can be changed to be `const` too. I will update them in this PR.